src/js/api/callbacks-deprecate.js

```diff
@@ -11,6 +11,7 @@ define([
             onFullscreen: events.JWPLAYER_FULLSCREEN,
             onMeta: events.JWPLAYER_MEDIA_META,
             onMute: events.JWPLAYER_MEDIA_MUTE,
+            onPlaybackRateChange: events.JWPLAYER_PLAYBACK_RATE_CHANGED,
```

Comment:
This is for backwards support only. Do not add new methods to the list of ones we intend to deprecate.